package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "chat-ui",
-	"version": "0.6.0",
+	"version": "0.7.0",
```

Comment:
Not sure we want to bump the version just for this, wdyt @nsarrazin ?

Reply:
downgraded!